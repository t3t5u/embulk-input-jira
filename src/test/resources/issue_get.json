{
    "primitive": 1,
    "string": "string",
    "emptyObject": {},
    "object": {
        "x": 1
    },
    "null": null,
    "emptyArray": [],
    "primitiveArray": [
        1,
        2,
        3
    ],
    "nested": {
        "primitive": 1,
        "object": {},
        "array": [
            1,
            {},
            3
        ]
    },
    "allJsonObjectArray": [
        {
            "primitive": 1,
            "string": "string1",
            "only1": "only1"
        },
        {
            "primitive": 2,
            "string": "string2",
            "only2": "only2"
        }
    ]
}