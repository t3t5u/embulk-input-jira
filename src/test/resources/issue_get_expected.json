{
    "primitive": "primitive",
    "primitiveResult": 1,
    "string": "string",
    "stringResult": "string",
    "emptyObject": "emptyObject",
    "emptyObjectResult": {},
    "object": "object",
    "objectResult": {
        "x": 1
    },
    "null": "null",
    "nullResult": null,
    "notExisted": "notExisted",
    "notExistedResult": null,
    "notExisted2Levels": "notExisted.notExisted",
    "notExisted2LevelsResult": null,
    "emptyArray": "emptyArray",
    "emptyArrayResult": [],
    "primitiveArray": "primitiveArray",
    "primitiveArrayResult": [
        1,
        2,
        3
    ],
    "nestedPrimitive": "nested.primitive",
    "nestedPrimitiveResult": 1,
    "nestedObject": "nested.object",
    "nestedObjectResult": {},
    "nestedArray": "nested.array",
    "nestedArrayResult": [
        1,
        {},
        3
    ],
    "3levelsArraysNull": "nested.array.null",
    "3levelsArraysNullResult": [
        1,
        null,
        3
    ],
    "allJsonObjectArrayPrimitive": "allJsonObjectArray.primitive",
    "allJsonObjectArrayPrimitiveResult": [
        1,
        2
    ],
    "allJsonObjectArrayString": "allJsonObjectArray.string",
    "allJsonObjectArrayStringResult": [
        "string1",
        "string2"
    ],
    "allJsonObjectArrayOnly1": "allJsonObjectArray.only1",
    "allJsonObjectArrayOnly1Result": [
        "only1",
        null
    ],
    "allJsonObjectArrayOnly2": "allJsonObjectArray.only2",
    "allJsonObjectArrayOnly2Result": [
        null,
        "only2"
    ]
}